scripts/adaptBuildToFf.mts

```diff
@@ -0,0 +1,25 @@
+import fs from 'fs/promises';
```

Comment:
Not sure if it matters, but [docs](https://nodejs.org/docs/latest-v16.x/api/fs.html#fspromisesreadfilepath-options) import like this
```
import * as fs from 'node:fs/promises';
```
or 
```
import { readFile, writeFile } from 'node:fs/promises';
```

Reply:
I don't think it does.